wgpu-core/src/command/render.rs

```diff
@@ -131,6 +131,30 @@ pub enum RenderCommand {
         buffer_id: id::BufferId,
         offset: BufferAddress,
     },
+    MultiDrawIndirect {
+        buffer_id: id::BufferId,
+        offset: BufferAddress,
+        count: u32,
```

Comment:
We should also probably merge this with `DrawIndirect` by having `count: Option<NonZeroU32>`